### ISSUE TITLE
v0.3.5+cmake 3.90+vs2017 Compile error

### ISSUE THREAD

**Issue by hurricanemad** [xianyi/OpenBLAS#2065]
I compiled the version 0.3.5 on visual studio 2017 many times. When I compiled the openblas and lapack with Intel fortran compiler , it failed with following prompt:
 10>Couldn't open file 'D:/OpenBLAS/myopenblas/openblas.dir/Debug/lapacke_sbbcsd.obj' with CreateFile()
10>系统找不到指定的批处理标签 - VCEnd
10>openblas : error PRJ0019: A tool returned an error code

The build log have uploaded. Please help me and give some tips.
[BuildLog.htm.pdf](https://github.com/xianyi/OpenBLAS/files/2984778/BuildLog.htm.pdf)

I deleted the msys2 in system path and installed the Perl64.

**Comment by martin-frbg** [comment]
Unfortunately the build log you uploaded appears to be incomplete - the long lines appear to be truncated to the screen (or default paper ?) size. Even more importantly I do not see anything related to the compilation of lapacke_sbbcsd.c or any of the other C files, just the FORTRAN compiler (ifort)  Did you start this build from a clean directory ?

**Comment by hurricanemad** [comment]
Thank for your replying. Sorry, I rebuild the sln and record the log in buildlog. It's disgusting thing to not find the information of lapacke_sbbcsd.c. I start build from a clean directory with Cmake building sln files. I apprieciate to anyone give me some tips.
[buildlog.zip](https://github.com/xianyi/OpenBLAS/files/2986477/buildlog.zip)

**Comment by martin-frbg** [comment]
Sorry, that log now contains lots of data from the cmake and/or Visual Studio environment but even less detail of the actual compiler run than before.

**Comment by brada4** [comment]
No compiler gets invoked, just linker error...
I see only this:
https://docs.microsoft.com/en-us/cpp/error-messages/tool-errors/linker-tools-error-lnk1104?view=vs-2017
```
11>LINK : fatal error LNK1104: cannot open file '..\lib\DEBUG\openblas.lib'
```
Is there openblas.lib built at that point?

**Comment by martin-frbg** [comment]
Well, it had better invoke the compiler at some (earlier) point to create the obj. Your very first log had at least some lines where ifort was compiling the fortran parts, not sure what you did differently then, but without the compiler output it is very hard to guess what could have happened. The LNK1104 is probably just a later consequence of earlier build errors, it may be trying to link the test cases although the library had not been created due to the missing obj.

**Comment by brada4** [comment]
There is no compiler invoked, just verbose env dumps from high debug level in vs2017

**Comment by martin-frbg** [comment]
Hopefully fixed now in 0.3.6 (based on comment from #2091).

**Comment by hurricanemad** [comment]
> Hopefully fixed now in 0.3.6 (based on comment from #2091).

Thanks very much! That's awesome. I'll try it!
